02-express-tutorial/answers/routes/auth.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const router = express.Router();
+
+router.post('/', (req, res) => {
+	const { name } = req.body;
+	!!name ? res.status(200).send(`Welcome ${name}`) : null;
```

Comment:
Minor comment: While this is a correct use of the `ternary` statement in Javascript; you may find that in a lot of codebases, it is not considered good practice to use a ternary statement where you're not actually assigning the return value to anything. Eg: in this case, the `else` part (after the colon `:` symbol) isn't doing anything, just returning `null`. It would be clearer to just stick to an `if` statement in this case since we don't need the `else` part to do anything. But overall, this is just up to preferences.

Reply:
Also, this may not work the way you expect, since on line 6, we are not calling `return`. So the javascript engine will always continue to read all the way to line 7, which means you will see something like the following error in your console when running the node server: 
```
Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
```

Reply:
Oh got it! Thank you!